packages/block-library/src/post-content/index.js

```diff
@@ -15,6 +15,9 @@ export { metadata, name };
 
 export const settings = {
 	title: _x( 'Post Content', 'block title' ),
+	description: __(
+		'Displays the contents of a post, without title or metadata like tags or categories.'
```

Comment:
maybe: "of a post or page". I don't think the "without the title..." is needed.

Reply:
Just this then? 'Displays the contents of a post or page.'